src/common/dcsvg.cpp

```diff
@@ -313,10 +490,27 @@ void wxSVGFileDCImpl::DoDrawLine (wxCoord x1, wxCoord y1, wxCoord x2, wxCoord y2
 
 void wxSVGFileDCImpl::DoDrawLines(int n, const wxPoint points[], wxCoord xoffset , wxCoord yoffset )
 {
-    for ( int i = 1; i < n; i++ )
+    if (n > 1)
     {
-        DoDrawLine ( points [i-1].x + xoffset, points [i-1].y + yoffset,
-            points [ i ].x + xoffset, points [ i ].y + yoffset );
+        NewGraphicsIfNeeded();
+        wxString s;
+
+        s = wxT("<path ") + wxGetPenPattern(m_pen);
+        s += wxString::Format(wxS("d=\"M%d %d "), (points[0].x + xoffset), (points[0].y + yoffset));
+        CalcBoundingBox(points[0].x + xoffset, points[0].y + yoffset);
+
+        for (int i = 1; i < n; ++i)
+        {
+            s += wxString::Format(wxS("L%d %d "), (points[i].x + xoffset), (points[i].y + yoffset));
+            CalcBoundingBox(points[i].x + xoffset, points[i].y + yoffset);
+        }
+
+        s += wxS("\" style=\"fill:none\" /> \n");
```

Comment:
Is this space between XML tag and EOL really necessary? These trailing spaces are used inconsistently in the code.